### ISSUE TITLE
stop the generation of reports other than index.html

### ISSUE THREAD

**Issue by shrutigupta123** [serenity-bdd/serenity-cucumber-starter#104]
I want to reduce the size of my target folder, for that purpose I don't want all the serenity reports to be generated other than index.html, Is there a way to achieve this?

**Comment by wakaleo** [comment]
No, there is not way of doing that currently. But you can generate the single page HTML report which is self-contained.

**Comment by shrutigupta123** [comment]
How can I generate a single page HTML report?

**Comment by shrutigupta123** [comment]
Also my main goal is to **reduce the size of the target folder**, so will the generation of single page HTML report help with that?
